chwifi

```diff
@@ -85,6 +85,34 @@ update_routine() {
     fi
 }
 
+update_profiles() {
+    printf '%s\n' "Updating profiles..."
+    config=$(config_dir)
+    profiles=""
+    # Grab other profiles and concatenate in string, using semicolon as seperator
+    for entry in "/etc/netctl"/*; do
+        if [ -f "$entry" ]; then
+            profiles+=$(echo "$entry" | sed -e 's/\/etc\/netctl\///g')';'
+        fi
+    done
+    # Remove final semicolon from string
+    profiles=${profiles::-1}
+    # Regex other profiles into config
+    sed -i "s/\"$network_manager_other_profiles\"$/\"$profiles\"/" "$config/config"
+}
+
+check_profile() {
```

Comment:
This function could do with some commenting for further referencing

Reply:
8e780dd fixes this.